lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
The important thing is to make sure that different production environments don't share the same Skylight token. The steps for setting up staging are documented here (https://docs.skylight.io/getting-set-up/#setting-up-multiple-environments) and, as we're well aware, are less than ideal. given the extra effort to set up another environment it probably makes sense to not have it on by default.

That said, we are currently working on some stuff that should make multiple environments much more streamlined. Once this all lands then I think it makes perfect sense to enable staging by default as we'll be able to track separate environments with one token.